k8s/deploy

```diff
@@ -226,16 +226,20 @@ isAllowedDeployEnvironment() {
 }
 
 configureSlackNotifications() {
-    EMOJIS=(":celebrate:"  ":party_dinosaur:" ":hammer-time:"  ":andela:" ":victory-danch:" ":aw-yeah:" ":carlton-dance:" ":yes:" ":partyparrot:" ":dancing-penguin:" ":aww-yeah-remix:" )
+    # declare any 10 emojis 
+    # for example; EMOJIS=(":celebrate:"  ":party_dinosaur:" ":hammer-time:"  ":andela:" ":victory-danch:" ":aw-yeah:" ":carlton-dance:" ":partyparrot:" ":dancing-penguin:" ":aww-yeah-remix:" )
+    EMOJIS=(${EMOJI_1}  ${EMOJI_2} ${EMOJI_3}  ${EMOJI_4} ${EMOJI_5} ${EMOJI_6} ${EMOJI_7} ${EMOJI_8} ${EMOJI_9} ${EMOJI_10} )
```

Comment:
Instead of making user add so many emojis, you as them to pass an array of them. And assign default is not specified.
```bash

configureSlackNotifications() {
    # get the user emojis
    DEFAULT_EMOJIS=(
        ":celebrate:" 
        ":party_dinosaur:"
        ":hammer-time:" 
        ":andela:" 
        ":victory-danch:" 
        ":aw-yeah:" 
        ":carlton-dance:"
        ":partyparrot:" 
        ":dancing-penguin:"
        ":aww-yeah-remix:" 
    )
    local __emojis=$1

    if [ "$__emojis" ]; then
        EMOJIS=$__emojis
    else
        EMOJIS=$DEFAULT_EMOJIS
    fi
    
    # your logic goes here
}
```